test/types/reply.test-d.ts

```diff
@@ -32,6 +34,11 @@ const getHandler: RouteHandlerMethod = function (_request, reply) {
   expectType<(payload: any) => string | ArrayBuffer | Buffer>(reply.serialize)
   expectType<(fulfilled: () => void, rejected: (err: Error) => void) => void>(reply.then)
   expectType<FastifyInstance>(reply.server)
+  expectAssignable<((httpStatus: string) => DefaultSerializationFunction)>(reply.getSerializationFunction)
+  expectAssignable<((schema: {[key: string]: unknown}) => DefaultSerializationFunction)>(reply.getSerializationFunction)
+  expectAssignable<((schema: {[key: string]: unknown}, httpStatus?: string) => DefaultSerializationFunction)>(reply.compileSerializationSchema)
+  expectAssignable<((input: {[key: string]: unknown}, schema: {[key: string]: unknown}, httpStatus?: string) => unknown)>(reply.serialize)
+  expectAssignable<((input: {[key: string]: unknown}, httpStatus: string) => unknown)>(reply.serialize)
```

Comment:
```suggestion
  expectAssignable<((input: {[key: string]: unknown}, httpStatus: string) => unknown)>(reply.serializeInput)
```

Reply:
Ah, see what you mean. Huge typo, thanks! 🙏

Reply:
Fixed on [a968a35](https://github.com/fastify/fastify/pull/4158/commits/a968a354eb21fa45593bd9a4f2c807962bb5d69d). Can you take a look? :)